CMakeLists.txt

```
@@ -43,6 +43,7 @@ endif()
 list(PREPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
 include(project_options)
+include(cmake/unity.cmake)
```

We can keep it simple:

```suggestion
include(unity)
```